package.xml

```diff
@@ -67,7 +67,7 @@
       <package>
         <name>psr</name>
         <channel>pecl.php.net</channel>
-        <min>0.7.0</min>
+        <min>1.0.0</min>
```

Comment:
Can we leave this at 0.7? I don't force people to update to 1.0 on a maintenance release. We can do this in 4.1.
0.7 is btw the same source as 1.0.